package.json

```diff
@@ -69,7 +71,9 @@
 		"test-modules": "NODE_ENV=test NODE_PATH=tests:_inc/client tests/runner.js modules",
 		"test-dangerci-and-adminpage-and-extensions": "yarn danger ci && yarn test-adminpage && yarn test-extensions",
 		"test-search": "jest modules/search",
-		"reformat-files": "./node_modules/.bin/prettier --ignore-path .eslintignore --write \"**/*.{js,jsx,json}\"",
+		"reformat-files": "npm-run-all --parallel --continue-on-error reformat-files:*",
+		"reformat-files:scss": "npm run -s lint:scss -- --fix || exit 0",
```

Comment:
Instead of `npm run -s lint:scss -- --fix` I could maybe run `yarn lint:scss --fix`?